src/Middleware/SpaServices.Extensions/src/AngularCli/AngularCliMiddleware.cs

```diff
@@ -48,27 +48,19 @@ internal static class AngularCliMiddleware
             var diagnosticSource = appBuilder.ApplicationServices.GetRequiredService<DiagnosticSource>();
             var angularCliServerInfoTask = StartAngularCliServerAsync(sourcePath, scriptName, pkgManagerCommand, devServerPort, logger, diagnosticSource, applicationStoppingToken);
 
-            // Everything we proxy is hardcoded to target http://localhost because:
-            // - the requests are always from the local machine (we're not accepting remote
-            //   requests that go directly to the Angular CLI middleware server)
-            // - given that, there's no reason to use https, and we couldn't even if we
-            //   wanted to, because in general the Angular CLI server has no certificate
-            var targetUriTask = angularCliServerInfoTask.ContinueWith(
-                task => new UriBuilder("http", "localhost", task.Result.Port).Uri);
-
             SpaProxyingExtensions.UseProxyToSpaDevelopmentServer(spaBuilder, () =>
             {
                 // On each request, we create a separate startup task with its own timeout. That way, even if
                 // the first request times out, subsequent requests could still work.
                 var timeout = spaBuilder.Options.StartupTimeout;
-                return targetUriTask.WithTimeout(timeout,
+                return angularCliServerInfoTask.WithTimeout(timeout,
                     $"The Angular CLI process did not start listening for requests " +
                     $"within the timeout period of {timeout.TotalSeconds} seconds. " +
                     $"Check the log output for error information.");
             });
         }
 
-        private static async Task<AngularCliServerInfo> StartAngularCliServerAsync(
+        private static async Task<Uri> StartAngularCliServerAsync(
```

Comment:
The `AngularCliServerInfo` class seems to be unused now and could be removed.

Reply:
I have just removed it.

Reply:
This fix isn't working as reproduced with the code of this PR in this repo:

https://github.com/asadsahi/dotnet-angular11